Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/MethodConfiguration.cpp

```diff
@@ -56,7 +56,7 @@ namespace ScriptCanvas
             {
                 const Data::Type outputType = (unpackedTypes.size() == 1 && AZ::BehaviorContextHelper::IsStringParameter(*result)) ? Data::Type::String() : Data::FromAZType(unpackedTypes[resultIndex]);
 
-                AZStd::string resultSlotName(AZStd::string::format("Result: %s", Data::GetName(outputType).data()));
+                AZStd::string resultSlotName(Data::GetName(outputType).data());
```

Comment:
The `.data()` isn't needed.

This can just be 
```suggestion
                AZStd::string resultSlotName(Data::GetName(outputType));
```